src/main/java/com/google/devtools/build/lib/bazel/Bazel.java

```diff
@@ -76,7 +76,8 @@
           com.google.devtools.build.lib.metrics.PostGCMemoryUseRecorder.GcAfterBuildModule.class,
           com.google.devtools.build.lib.packages.metrics.PackageMetricsModule.class,
           com.google.devtools.build.lib.metrics.MetricsModule.class,
-          BazelBuiltinCommandModule.class);
+          BazelBuiltinCommandModule.class,
+          com.google.devtools.build.lib.includescanning.IncludeScanningModule.class);
```

Comment:
Sure; it's not clear what the style of this `BLAZE_MODULES` list is supposed to be. Most but not all of the entries are fully-qualified.

Reply:
I see, I didn't notice the others. importing is the right way according to our presubmit checks. You can leave the others above as they are since you didn't touch those.